pos_anonymous_customer/models/res_partner.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    available_in_pos = fields.Boolean(
+        string='Available for POS',
```

Comment:
@legalsylvain The default is quite deliberately to not load the customer when not specifically set. We have a project with > 132.000 customers (they also have a webshop). Loading the customer data takes minutes at the physical shop they have, and is never used.

If a partner is no longer a customer, he/she will not be loaded in the POS anyway, as in the end I did not replace the default domain - [['customer','=',true]] - but appended the check for available_in_pos.

I will rename the module according to your suggestion.

Also the ideas for a more flexible approach are good, but indeed not (yes?) in the scope of what we need right now.